Fix name in protocolAny

In [documentation](https://docs.openstack.org/python-openstackclient/latest/cli/command-objects/security-group-rule.html#cmdoption-openstack-security-group-rule-create-protocol): 
valid value for openstack cli is 'any'. In fact it's converted in to `None` and then in `null`:


debug from openstack call:

`openstack security group rule create --egress --remote-ip 8.8.8.8/32 --protocol 'any' eece1c60-baf3-4ea9-88bf-54cc40290720`

`REQ: curl -g -i -X POST http://test:9696/v2.0/security-group-rules -H "Content-Type: application/json" -H "User-Agent: openstacksdk/0.99.0 keystoneauth1/4.6.0 python-requests/2.28.1 CPython/3.9.13" -H "X-Auth-Token: {SHA256}547fc1809352d6d9431590520fb0923bbf92d25fbe73af21eb488e1b71e67e52" -d '{"security_group_rule": {"security_group_id": "eece1c60-baf3-4ea9-88bf-54cc40290720", "ethertype": "IPv4", "protocol": null, "direction": "egress", "remote_ip_prefix": "8.8.8.8/32"}}'`

OK I may have found the problem you want to fix.

I have tried creating a group and a rule with this code:

```Go
	group, err := groups.Create(ctx, networkClient, groups.CreateOpts{
		Name:        "pprinett-test",
		Description: "delete me",
	}).Extract()
	if err != nil {
		panic(err)
	}

	rule1, err := rules.Create(ctx, networkClient, rules.CreateOpts{
		Direction:    rules.DirEgress,
		Description:  "delete me",
		EtherType:    rules.EtherType4,
		SecGroupID:   group.ID,
		PortRangeMax: 4000,
		PortRangeMin: 4000,
		Protocol:     rules.ProtocolAny,
	}).Extract()
	if err != nil {
		panic(err)
	}
```

And what I got is this error message:
```
panic: Expected HTTP response code [201 202] when accessing [POST https://neutron.example:13696/v2.0/security-group-rules], but got 400 instead: {"NeutronError": {"type": "SecurityGroupRuleInvalidProtocol", "message": "Security group rule protocol any not supported. Only protocol values [None, 'ah', 'dccp', 'egp', 'esp', 'gre', 'hopopt', 'icmp', 'igmp', 'ip', 'ipip', 'ipv6-encap', 'ipv6-frag', 'ipv6-icmp', 'icmpv6', 'ipv6-nonxt', 'ipv6-opts', 'ipv6-route', 'ospf', 'pgm', 'rsvp', 'sctp', 'tcp', 'udp', 'udplite', 'vrrp'] and integer representations [0 to 255] are supported.", "detail": ""}}
```

--- a/openstack/networking/v2/extensions/security/rules/requests.go
+++ b/openstack/networking/v2/extensions/security/rules/requests.go
@@ -75,7 +75,7 @@ const (
 	ProtocolUDP       RuleProtocol  = "udp"
 	ProtocolUDPLite   RuleProtocol  = "udplite"
 	ProtocolVRRP      RuleProtocol  = "vrrp"
-	ProtocolAny       RuleProtocol  = "any"
+	ProtocolAny       RuleProtocol  = ""
 )
 
 // CreateOptsBuilder allows extensions to add additional parameters to the

